### ISSUE TITLE
problem with systemd-login, doesn't start the display server

### ISSUE THREAD

**Issue by Stef9998** [systemd/systemd#11413]
Problem occured when upgrading from 239 -> 240

After boot it shows a black screen with an Underscore. Then after some time the Underscore start blinking and not until then I can switch to a tty. Before that the system just responds to nothing.

At tty I ran "journalctl -p 3 -xb" and got this output:
```
-- Logs begin at Sun 2019-01-13 08:01:35 CET, end at Sun 2019-01-13 08:58:55 CET. --
Jan 13 08:43:20 stefan-desk-arch NetworkManager[614]: <error> [1547365400.9559] session-monitor: failed to create systemd-logind monitor: -2
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.Shell.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.XSettings.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.Clipboard.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.Keyboard.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.Color.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.Power.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.MediaKeys.desktop
Jan 13 08:44:51 stefan-desk-arch gnome-session-binary[674]: Unrecoverable failure in required component org.gnome.SettingsDaemon.Wacom.desktop
```

Then when I restart systemd-logind with "systemctl restart systemd-logind". It takes a second and then takes me to my loginscreen from gdm how it normally should be at boot.

Downgrade to 239 fixes the issue
system: Arch Linux
display manager: gdm

could be same as #11299 but not sure

**Comment by yuwata** [comment]
I guess this is a duplicate of #11293, instead of #11299. See also https://bugzilla.redhat.com/show_bug.cgi?id=1662857.

Anyway, the issues are already fixed. Please try the current git snapshot.

**Comment by stef9998** [comment]
How can I install the current snapshot?
I can't really find anything on how to build it.

**Comment by tomty89** [comment]
@stef9998 build this with makepkg: https://aur.archlinux.org/packages/systemd-git/ (You can add `#commit=someid` to https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=systemd-git#n21 beforehand if you want to test a specific snapshot). See the man page (and perhaps the archwiki page) of makepkg.

**Comment by tomty89** [comment]
@stef9998 it will build the latest commit if you don't specify one, the version would be updated when you run makepkg, so check with `pacman -Qs systemd` for the exact version you got.

**Comment by poettering** [comment]
So it appears this is already fixed in v241, as suggested by https://github.com/systemd/systemd/issues/11413#issuecomment-453849847. Let's hence close this. Please report back if this is reproducible on v241.

**Comment by stef9998** [comment]
But after that fix was already in the newest snapshot, I build it and it still didn't work for me as mentioned there: https://github.com/systemd/systemd/issues/11413#issuecomment-454067116 
I will try the newest version when I have time to fix my system afterwards, in case it's still broken

**Comment by stef9998** [comment]
Finished my exams, tried to update today, bug is still happening.
systemd: 241.67-1

I'm going to be away from my desktop for 3 weeks in some hours, so I can't try and test stuff.
But you could tell me which logs I should provide for you when I come back.

Sorry for the complications, hope it gets fixed soon.

**Comment by jvschiavo** [comment]
I specifically have the error:
`NetworkManager[6160]: <error> [1777362086.8820] session-monitor: failed to create systemd-logind monitor: -2`
